src/main/java/com/autotune/analyzer/recommendations/term/Terms.java

```diff
@@ -114,6 +115,80 @@ public static boolean checkIfMinDataAvailableForTerm(ContainerData containerData
         return false;
     }
 
+    public static boolean checkIfMinDataAvailableForTermForNamespace(NamespaceData namespaceData,
+                                                                     Terms term,
+                                                                     Timestamp monitoringEndTime,
+                                                                     double measurementDuration) {
+
+        // Check if data is available
```

Comment:
This piece of code looks redundant. Can we have a common method for this which can accommodate both for Term and Namespace ?

Reply:
As discussed, we will address it later.